virttest/test_setup.py

```diff
@@ -1465,6 +1466,46 @@ def is_binded_to_stub(self, full_id):
         stub_path = os.path.join(base_dir, "drivers/%s" % self.device_driver)
         return os.path.exists(os.path.join(stub_path, full_id))
 
+    def port_id_vf(self):
```

Comment:
`generate_ib_port_id()` would be better to name it.

Reply:
@balamuruhans Made the necessary changes.